models/meteora/meteora_trades.sql

```diff
@@ -0,0 +1,159 @@
+ {{
+  config(
+        schema = 'meteora',
+        alias = 'trades',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'tx_index','block_month'],
+        pre_hook='{{ enforce_join_distribution("PARTITIONED") }}',
+        post_hook='{{ expose_spells(\'["solana"]\',
```

Comment:
if the goal is to integrate this into solana dex trades, let's remove this post hook to expose and keep off data explorer?